src/Orleans.Core/Runtime/RingRange.cs

```diff
@@ -378,7 +364,6 @@ public static IRingRange GetEquallyDividedSubRange(IRingRange range, int numSubR
                     switch (multiRange.Ranges.Count)
                     {
                         case 0: return multiRange;
-                        case 1: return GetEquallyDividedSubRange(multiRange.Ranges[0], numSubRanges, mySubRangeIndex);
```

Comment:
I suppose this is dead code elimination and that we keep `case 0` since it represents an empty range